netket/machine/py_rbm.py

```diff
@@ -47,8 +47,10 @@ def __init__(
             raise ValueError("`alpha` should be non-negative")
         m = int(round(alpha * n))
         self._w = _np.empty([m, n], dtype=_np.complex128)
-        self._a = _np.empty(n, dtype=_np.complex128) if use_visible_bias else None
-        self._b = _np.empty(m, dtype=_np.complex128) if use_hidden_bias else None
+        self._a = _np.empty(
+            n, dtype=_np.complex128) if use_visible_bias else None
+        self._b = _np.empty(
+            m, dtype=_np.complex128) if use_hidden_bias else None
```

Comment:
What's up with the formatting? Did you by any change forget to run Black?

Reply:
Yes I had an issue with Black, now it should be fine!